### ISSUE TITLE
iOS: Unable to calibrate x axis on iPad

### ISSUE THREAD

**Issue by vit-azarov** [koush/vysor.io#1205]
**To Reproduce**
1. Installed Dongle (MakerDiary w. Case)
2. Launch Vysor
3. Connect ipad pro 11
4. Vysor recognized device
5. Select Play
6. Vysor window launched

**Expected behavior**
Vysor works fine with ipad. Calibration works as expected. 

**Actual behavior**
In reality I was able to calibrate on y axis (300 works good) but x axis is out and I am not able to adjust it. Also seems like auto calibration does not work.

**Screenshots**

https://user-images.githubusercontent.com/100236985/214305438-05dde859-95b0-4e55-be49-1b22789e6f44.mp4

**Desktop (please complete the following information):**
 - MacOS Monterey 12.5.1

**iOS (please complete the following information):**
 - iPad Pro 11-inch (3rd gen)
 - iPadOS 16.2
 - Vysor Bug Report Link: I tried to log a bug report, but not sure where should this be stored.

**Comment by koush** [comment]
try setting the initial values to 50 and then run auto calibration? the initial values may be really out of range from what I'm seeing.

**Comment by robertr64** [comment]
I have a similar issue. I tried the default calibration of 100 and 100 and then tried calibrating with the built-in Vysor tool, and in both cases, the Ipad can move the cursor around on the lower half of the screen, but not the upper half. Works fine on an Iphone 12 Pro. It seems like it is not seeing the full IPad screen size. Also, any way to jump from one home screen page to another directly from the computer without touching the IOS device?

**Comment by vit-azarov** [comment]
Yup, same behaviour. I ended up using ipad emulator from xcode. Using vysor for android only now. 
Tho would be happy if the issue addressed

> I have a similar issue. I tried the default calibration of 100 and 100 and then tried calibrating with the built-in Vysor tool, and in both cases, the Ipad can move the cursor around on the lower half of the screen, but not the upper half. Works fine on an Iphone 12 Pro. It seems like it is not seeing the full IPad screen size. Also, any way to jump from one home screen page to another directly from the computer without touching the IOS device?
